modules/apps/site-initializer/site-initializer-raylife/extra/direct-to-customer/craco.config.js

```diff
@@ -47,6 +47,13 @@ module.exports = {
 				],
 			});
 
+			/**
+			 * Avoid hashes in filenames
+			 */
+
+			webpackConfig.output.chunkFilename = 'static/js/[name].js';
+			webpackConfig.output.filename = 'static/js/[name].js';
```

Comment:
@kevenleone, would this be alright, or mess up something in development mode?

Reply:
Shouldn't impact development mode, just the output of resources after build.

But, it's looks good!